include/deal.II/lac/full_matrix.h

```diff
@@ -995,12 +1007,24 @@ class FullMatrix : public Table<2, number>
    * @param A The first matrix (m x n).
    * @param B The second matrix (p x q).
    * @param adding If `true`, the result is added to the current matrix. If
-   *               `false` (default), the current matrix is overwritten.
+   *               `false`, the current matrix is overwritten.
    */
   void
   kronecker_product(const FullMatrix<number> &A,
                     const FullMatrix<number> &B,
-                    const bool                adding = false);
+                    const bool                adding);
+
+
+  /**
+   *  Compute the Kronecker product of multiple matrices. The adding parameter
+   *  is set to false.
```

Comment:
This documentation is referencing a parameter `adding` that exists in another function. Can you explain this better?

Separately, the documentation here should say what the `Rest...` types correspond to. Perhaps even give an example.